M2/Macaulay2/packages/Msolve.m2

```diff
@@ -389,33 +389,26 @@ Node
 	    value is also represented by a two element list of rational numbers, {a, b},
 	    this means that that coordinate of the solution has a value greater than or
 	    equal to a and less than or equal to b. This interval is computed symbolically
-	    and its correctness is guaranteed by exact methods. Note that using the option
-	    Output one may specify the output in terms of either a float interval with
-	    "floatInterval" or an average of the interval endpoints as a single float with "float".
-
+	    and its correctness is guaranteed by exact methods.
+	Text
+	    Option "output type" (default = "rational interval") gives alternatives ways to provide output 
+	    eitherusing @TO RRi@ ("float interval")
```

Comment:
```suggestion
	    either using @TO RRi@ ("float interval")
```

Reply:
@fchapoton I'm surprised codespell didn't catch this. Do you know what happened?